CONTRIBUTING.md

```diff
@@ -93,12 +93,13 @@ To run tests locally, please read the [CI instructions](ci/README.md).
   as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 * Make sure [CLA](https://identity.linuxfoundation.org/projects/cncf) is
   signed and CI is clear.
+* For non-trivial changes, please update the [CHANGELOG](./CHANGELOG.md).
 
 ### How to Get PRs Merged
 
 A PR is considered to be **ready to merge** when:
 
-* It has received two approvals from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
+* It has received atleast one approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
```

Comment:
```suggestion
* It has received at least one approval from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
```

Reply:
Also should we require at least two approvals? Perhaps one approval for trivial change is fine.

Reply:
Good point. Let's hear from others before finalizing it.

Reply:
OK, I have updated guideline to have
 - One approval ( from different company ) for PRs raised by approver / maintainers.
- Two approvals with one approval from different company for all other PRs.
Let me know if that will work ?